isvcs/es.go

```diff
@@ -77,9 +77,9 @@ func initElasticSearch() {
 		Timeout:     DEFAULT_HEALTHCHECK_TIMEOUT,
 	}
 
-	healthChecks := map[string]healthCheckDefinition{
-		DEFAULT_HEALTHCHECK_NAME: defaultHealthCheck,
-	}
+	healthChecks := make([]map[string]healthCheckDefinition, 1)
+	healthChecks[0] = make(map[string]healthCheckDefinition)
+	healthChecks[0][DEFAULT_HEALTHCHECK_NAME] = defaultHealthCheck
```

Comment:
Just FYI - if you wanted to use a static declaration, something like this would also work:
```
healthChecks := []map[string]healthCheckDefinition{
    map[string]healthCheckDefinition{
        DEFAULT_HEALTHCHECK_NAME: defaultHealthCheck,
    }
}
```
Which is actually 1 line longer, so potaytoe, potahtoe